README.md

```diff
@@ -422,6 +422,8 @@ require 'gumboot/shared_examples/database_schema'
 
 RSpec.describe 'Database Schema' do
   let(:connection) { ActiveRecord::Base.connection.raw_connection }
+  # Use the following (as an example) for column based exemptions
+  let(:collation_exemptions) { [%i[table_name column_name]] }
```

Comment:
Why arrays when you could do this?

```ruby
let(:collation_exemptions) do
  {
    objects: %i[col1 col2 col3],
    child_objects: %i[col4 col5 col6]
  }
end
```

The compact example is nice when it's a single column, and nasty as it grows imo.

```ruby
let(:collation_exemptions) do
  [
    %i[objects col1],
    %i[objects col2],
    %i[objects col3],
    %i[child_objects col4],
    %i[child_objects col5],
    %i[child_objects col6]
  ]
end
```